### ISSUE TITLE
Screen margin in all modes

### ISSUE THREAD

**Issue by houqp** [koreader/kindlepdfviewer#115]
Refer to http://www.mobileread.com/forums/showpost.php?p=2042845&postcount=239

**Comment by vdp** [comment]
Hi,
I think there will be different opinions about the margins and scrollbars. It depends on the particular document, but in many cases I for one wouldn't like to loose precious screen space for margins or non-essential UI elements. Ideally this could be a configurable option.

**Comment by houqp** [comment]
Yes, completely agree with you. It must be configurable, otherwise, we will end up to be another sucking reader like the native one.

**Comment by dracodoc** [comment]
I support the configurable margin option. The current design did use every last pixel, but the edge could be shadowed by the screen holder.

I also would like to have the option to make the margin border invisible. Right now the pdf often have a gray border in fit page view and other views. I think these gray areas are distracting and useless. The original pdf reader will just display them as invisible. If needed, the border can be displayed with small cross at four corners.

**Comment by eLiNK2gl** [comment]
just a small extension.
If the ratio of the height vs. width for the content is greater than that of the display, you automatically would get some margins horizontally. The only issue  is that the page is currently placed in the top left corner.
Centering it horizontally might be a better option.
